docs/rules/no-useless-constructor.md

```diff
@@ -0,0 +1,56 @@
+# Disallow unnecessary constructor (no-useless-constructor)
+
+ES2015 provides a default constructor if one is not specified. As such, it is unnecessary to provide an empty constructor or one that simply delegates into its parent class.
+
+## Rule Details
+
+This proposal is to provide a rule to warn against this type of redundant dead code so it can be removed. This rule is similar to `no-empty`.
+
+The following patterns are considered problems:
+
+```js
+/*eslint no-useless-constructor: 2*/
+/*eslint-env es6*/
```

Comment:
@platinumazure yes, this still works. 

@alberto http://eslint.org/docs/2.0.0/user-guide/migrating-to-2.0.0#language-options